modules/well-known.yml

```
@@ -0,0 +1,15 @@
+name: well-known
+description: A Nuxt.js module to add well-known URIs using middlewares
+repo: @zadigetvoltaire/nuxt-well-known
```

Should this be as below?

```suggestion
repo: zadigetvoltaire/nuxt-well-known
```